flaskapp/app/routes.py

```diff
@@ -116,96 +118,121 @@ def callback():
         
 ## Functions
 
-def get_venue_info(venue_name=None, venue_city=None, venue_id=None):
-    """Search for venue and retrieve info from Songkick.
-    To handle duplicate venue names, this function will check that venue's city matches that entered in venues.csv""" 
+def seatgeek_request(endpoint, venue_id = None, params = None):
+    """Format a request to SeatGeek API"""
+    parameters = {'client_id': SEATGEEK_CLIENT_ID,
+                 'client_secret': SEATGEEK_CLIENT_SECRET}
+    if params is not None:
+        for key, value in params.items():
+            parameters[key] = value
+    if venue_id is not None and endpoint == 'venues':
```

Comment:
Personally, I would split the different seatgeek API options out into their own smaller functions  (like `get_venue` or `get_events`) and construct the URL and parameter objects in those. They could then of course pass that info to a generic `seatgeek_request` function that handles everything else. That way you don't end up with a long list of conditional logic as your requests to seatgeek grow.